.rubocop.yml

```diff
@@ -138,6 +138,8 @@ Style/StringLiterals:
   EnforcedStyle: double_quotes
 Style/StringLiteralsInInterpolation:
   EnforcedStyle: double_quotes
+Style/SymbolArray:
+  Enabled: false
```

Comment:
This is an opinion and not a fact, this needs to be shipped as a unique pull request for everyone else to comment.